app/assets/stylesheets/application.scss

```diff
@@ -1,8 +1,18 @@
 @charset "utf-8";
 
-@import "normalize-rails";
-@import "bourbon";
-@import "base/grid-settings";
-@import "neat";
-@import "base/base";
-@import "refills/flashes";
+@import 'normalize-rails';
+@import 'bourbon';
+@import 'base/grid-settings';
+@import 'neat';
+@import 'base/base';
+@import 'refills/flashes';
+
+a {
+  .b {
+    .c {
+      .d {
+        color: blue;
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.